bin/test-dqlite-benchmark

```diff
@@ -73,5 +73,6 @@ sudo apt-get install -y dqlite-tools
 run_benchmark
 sleep 2
 run_benchmark --disk
+tar -zcf /home/ubuntu/artifacts.tar.gz dqlite-benchmark--disk.tar.gz dqlite-benchmark.tar.gz
```

Comment:
Is there a typo and should read `dqlite-benchmark-disk.tar.gz` instead (double hyphen)?

Reply:
The double hyphen is correct, I pass the argument to the script directly to the filename, it's maybe not super pretty.